### PR TITLE
Activityfeed follow

### PR DESCRIPTION
**Pull request by marouaneabra**
prototype

### CODE REVIEW

**Comment by fatlotus** [comment]
This is just UI changes, right?

File: src/main/webapp/WEB-INF/view/conversations.jsp [hunk 1]
```diff
@@ -13,77 +13,105 @@
...
+			%>
+		</ul>
+		<%
+		  }
+		%>
+		<hr />
+	</div>
 </body>
+<script>
+	function change()
```
Comment (fatlotus) [inline src/main/webapp/WEB-INF/view/conversations.jsp]: What does this function do?

File: src/main/java/codeu/controller/ConversationServlet.java [hunk 2]
```diff
@@ -118,5 +118,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     conversationStore.addConversation(conversation);
     response.sendRedirect("/chat/" + conversationTitle);
+
+    String convTest = request.getParameter("Following");
+    System.out.println("User entered conv name: " + conversationTitle);
+    if (convTest == null) {
+      System.out.println("it's null rn");
+    } else if (convTest.equals("Following")) {
+      System.out.print("Following rn");
```
Comment (fatlotus) [inline src/main/java/codeu/controller/ConversationServlet.java]: Might need a println, else the message won't show up (or be garbled)

File: src/main/webapp/WEB-INF/view/conversations.jsp (hunk 1 above)
Comment (marouaneabra) [reply src/main/webapp/WEB-INF/view/conversations.jsp]: Switches the button from Following to Follow after it's clicked

File: src/main/java/codeu/controller/ConversationServlet.java (hunk 2 above)
Comment (marouaneabra) [reply src/main/java/codeu/controller/ConversationServlet.java]: Still, convTest stays null even after I click the button

File: src/main/java/codeu/controller/ConversationServlet.java [hunk 3]
```diff
@@ -118,5 +118,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     conversationStore.addConversation(conversation);
     response.sendRedirect("/chat/" + conversationTitle);
+
+    String convTest = request.getParameter("Following");
```
Comment (ateitelbaum) [inline src/main/java/codeu/controller/ConversationServlet.java]: Where do you set the parameter "following"?

File: src/main/webapp/WEB-INF/view/conversations.jsp (hunk 1 above)
Comment (fatlotus) [reply src/main/webapp/WEB-INF/view/conversations.jsp]: Ah, I think that's triggered before the function is submitted. Maybe try removing this function?

**Pull request merged**